src/main/kotlin/no/nav/familie/oppdrag/repository/OppdragLagerRepositoryJdbc.kt

```diff
@@ -76,21 +76,23 @@ class OppdragLagerRepositoryJdbc(
     }
 
     override fun oppdaterStatus(oppdragId: OppdragId, oppdragStatus: OppdragStatus, versjon: Int) {
-        val update = "UPDATE oppdrag_lager SET status = '${oppdragStatus.name}' " +
-            "WHERE person_ident = '${oppdragId.personIdent}' " +
-            "AND fagsystem = '${oppdragId.fagsystem}' " +
-            "AND behandling_id = '${oppdragId.behandlingsId}'" +
-            "AND versjon = $versjon"
+        val update = "UPDATE oppdrag_lager SET status = ? " +
+            "WHERE person_ident = ? " +
+            "AND fagsystem = ? " +
+            "AND behandling_id = ?" +
```

Comment:
Må ha mellomrom etter ? for at det skal bli en gyldig sql

Reply:
Ja burde ju det, men testen ble grønn 👀